verification/curator-service/auth.md

```diff
@@ -0,0 +1,44 @@
+# Authentication and authorization
+
+## Authentication
+
+Users are currently authenticated using passport.JS's Google OAuth strategy.
```

Comment:
nit, because laziness: drop a link to the npm page?

Reply:
done